src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -163,7 +164,29 @@ export const ContactFlowColumn: React.FC<Props> = ({
     <Card>
       <ContainerBox p={2} data-testid="column-header" color={color}>
         <Box width="80%">
-          <ColumnTitle variant="h6">{title}</ColumnTitle>
+          {appealStatus === AppealStatusEnum.ReceivedNotProcessed ? (
+            <Tooltip
+              title={
+                <>
+                  <Typography>
+                    {t(
+                      'Do not move a contact into this column called "Received". Due to an outdated feature, contacts must first be moved to"Committed". If the gift has been recorded, you can then move the contact into the column called "Given"',
```

Comment:
And the same with the other label.

```suggestion
                      'Do not move a contact into this column called "Received". Due to an outdated feature, contacts must first be moved to "Committed". If the gift has been recorded, you can then move the contact into the column called "Given".',
```